### ISSUE TITLE
My printer sometimes prints pages containing "Invalid page range: Toner\\\ Cartrige\\\ S050147..."

### ISSUE THREAD

**Issue by chrta** [OpenPrinting/cups-filters#422]
My Espon C4100 printer connected via Ethernet sometimes prints, in addition to the correct pages, pages only containing two lines:

```
Invalid page range: Toner\\\ Cartrige\\\ S050147
                                                Invalid page range: Toner\\\
```

Is this the issue?
https://github.com/OpenPrinting/cups-filters/blob/master/filter/foomatic-rip/options.c#L2164

```
printf("Invalid page range: %s\n", tok);
```
Should this instead be a call to `_log()`?

Additional info:
I only found the toner cartrige strings that are printed in my /etc/cups/printers.conf:
```
...
MakeModel Epson AL-C4100 Foomatic/eplaser (recommended)
DeviceURI dnssd://C4100._printer._tcp.local/
...
Attribute marker-names Toner Cartridge S050148,Toner Cartridge S050147,Toner Cartridge S050146,Toner Cartridge S050149,Opc S051093,Fuser Unit S053011/S053012,Transfer Unit S053006
...
```

This may also be related to https://bugs.launchpad.net/ubuntu/+source/cups/+bug/1912911

**Comment by tillkamppeter** [comment]
Fixed in commit 32e60bb in master/2.x and commit 6906169 in 1.x.

Thanks for the bug report.

**Comment by tillkamppeter** [comment]
Fix is also included in today's 1.28.11 release of cups-filters.
